runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -91,6 +96,10 @@ XpTrackerConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(XpTrackerConfig.class);
 	}
 
+	@Getter(AccessLevel.PACKAGE)
+	public static final List<Skill> COMBAT = Arrays.asList(Skill.ATTACK, Skill.STRENGTH, Skill.DEFENCE, Skill.RANGED, Skill.HITPOINTS, Skill.MAGIC);
```

Comment:
Make this private or just package protected with no getter. Also use `ImmutableList` or set.